explainers/beforematch.md

```diff
@@ -288,6 +307,50 @@ scroll after adding an async step.
 5. **content-visibility: hidden-matchable -> visible**: find-in-page scrolls to
 to the revealed text.
 
+### Alternatives
+Given the purpose of displaying `content-visibility: hidden-matchable` text
+when it is searched for, there are a number of alternatives we have considered.
+
+#### Automatic Unlocking
+`content-visibility: hidden-matchable` text would be automatically changed to
+`content-visibility: visible` when the hidden text is searched for.
```

Comment:
This is not actually correct. We would keep an internal state that essentially ignores 'hidden-matchable'. The problem is that, unlike attributes, we can't change styles from within the browser. 

If we have something like
```
body > .randomClass {
  content-visibility: hidden-matchable;
}

it means that all first level children of body that have .randomClass have hidden-matchable. If we find one such element, we obviously can't change this style rule since it doesn't just apply to this element. We could change inline style of the element, but that sounds like a really bad approach since we don't know how script is using inline styles, so we shouldn't be messing with them.

Reply:
Thanks for the explanation, I resolved this in another thread